shopinvader_address_shipping_note/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import res_partner
```

Comment:
@AnizR can you move the shipping note model to another PR? Since it is just an example, I don't think we'll want to  merge it at the same time as shopinvader_address.

Reply:
Done: https://github.com/shopinvader/odoo-shopinvader/pull/1374